charts/garnet/README.md

```diff
@@ -1,6 +1,6 @@
 # garnet
 
-![Version: 0.1.2](https://img.shields.io/badge/Version-0.1.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.46](https://img.shields.io/badge/AppVersion-1.0.46-informational?style=flat-square)
+![Version: 0.1.3](https://img.shields.io/badge/Version-0.1.3-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.46](https://img.shields.io/badge/AppVersion-1.0.46-informational?style=flat-square)
```

Comment:
Does this 1.0.46 have to be manually updated very time we update Version.props with a new Garnet version?

Reply:
Actually yes because the `README.md` is generated by `helm-docs` taking the `README.md.gotmpl`.
I will should do a new PR to include this command and the git commit in the next few days